server/handler.js

```diff
@@ -20,3 +34,33 @@ module.exports.hello = async event => {
   // Use this code if you don't use the http event with the LAMBDA-PROXY integration
   // return { message: 'Go Serverless v1.0! Your function executed successfully!', event };
 };
+
+module.exports.createUser = async event => {
+  let obj = JSON.parse(event.body);
+  const user = User.build(obj);
```

Comment:
hmm is there any validation we should do before creating the user? for example checking if the user already exists?

Reply:
I put that in the models/user.js file, not sure that is enough or not

Reply:
ooh! yeah that's enough. I missed that